assets/sass/_event-list.scss

```diff
@@ -0,0 +1,57 @@
+.pulumi-event-list-container {
+    #all:target {
```

Comment:
You could cut down this duplication _even more_ with a [Sass @each](https://sass-lang.com/documentation/at-rules/control/each):

```
.pulumi-event-list-container {
    @each $filter-name in "all", "featured", "upcoming", "videos", "pulumitv" {
        ##{$filter-name}:target {
            ~ .event-list-filter li[data-filter-name="#{$filter-name}"] a {
                @apply text-blue-500;
            }

            ~ ul li[data-filters~="#{$filter-name}"] {
                @apply block;
            }
        }
    }
}
```

Reply:
TIL :)